test/integration/gcp/controls/forseti.rb

```diff
@@ -0,0 +1,136 @@
+# encoding: utf-8
+# copyright: 2018, The Authors
+
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+title 'Forseti Terraform GCP Test Suite'
+
+gcp_project_id = attribute('gcp_project_id')
+#gcp_org_id = attribute('gcp_org_id')
+
+control 'forseti-client-vm' do
+  impact 1.0
+  title 'Test forseti client vm'
+  describe google_compute_instances(project: gcp_project_id,  zone: 'us-central1-c') do
+    its('instance_names') { should include /forseti-client-vm-*/ }
+  end
+end
+
+control 'forseti-server-vm' do
+  impact 1.0
+  title 'Test foseti server vm'
+  describe google_compute_instances(project: gcp_project_id,  zone: 'us-central1-c') do
+    its('instance_names') { should include /forseti-server-vm-*/ }
+  end
+end
+
+control 'forseti-cloudsql-instance' do
+  impact 1.0
+  title 'Test CloudSQL Instance is up'
+  describe google_sql_database_instances(project: gcp_project_id) do
+    its('instance_names') { should include /forseti-server-db-*/ }
+  end
+end
+
+
+control 'forseti-server-iam-roles' do
+  gcp_enable_privileged_resources = '1'
+
+  impact 1.0
+  title 'Test Server Project IAM Role bindings'
+  describe google_project_iam_binding(project: gcp_project_id, role: "roles/storage.objectViewer") do
+    it { should exist }
+    its ('members'){ should include /forseti/ }
+  end
+  describe google_project_iam_binding(project: gcp_project_id, role: "roles/storage.objectCreator") do
+    it { should exist }
+    its ('members'){ should include /forseti/ }
+  end
+  describe google_project_iam_binding(project: gcp_project_id, role: "roles/cloudsql.client") do
+    it { should exist }
+    its ('members'){ should include /forseti/ }
+  end
+  describe google_project_iam_binding(project: gcp_project_id, role: "roles/cloudtrace.agent") do
+    it { should exist }
+    its ('members'){ should include /forseti/ }
+  end
+  describe google_project_iam_binding(project: gcp_project_id, role: "roles/logging.logWriter") do
+    it { should exist }
+    its ('members'){ should include /forseti/ }
+  end
+  describe google_project_iam_binding(project: gcp_project_id, role: "roles/iam.serviceAccountTokenCreator") do
+    it { should exist }
+    its ('members'){ should include /forseti/ }
+  end
+end
+
+# @TODO Need to figure out the Org level permissions to execute this test
```

Comment:
Avoid commented out code where possible. IMO having a pending test is better than a commented out test with a TODO, since the former is more visible and thus is more likely to be fixed.